client/src/services/models.ts

```diff
@@ -31,3 +31,5 @@ export interface CoursePageProps {
   session: Session;
   course: Course;
 }
+
+export type Action = 'expel' | 'unassign';
```

Comment:
can transfer to expelStudent.
or change the name of not just Action, but for what Action

Reply:
done